packages/fx-core/src/component/driver/aad/utility/aadAppClient.ts

```diff
@@ -99,6 +100,53 @@ export class AadAppClient {
     });
   }
 
+  public async getOwners(objectId: string): Promise<AadOwner[] | undefined> {
+    // const response = await instance.get(`${baseUrl}/applications/${objectId}/owners`);
+    const response = await this.axios.get(`applications/${objectId}/owners`, {
+      "axios-retry": {
+        retries: this.retryNumber,
+        retryDelay: axiosRetry.exponentialDelay,
+        retryCondition: (error) =>
+          axiosRetry.isNetworkError(error) ||
+          axiosRetry.isRetryableError(error) ||
+          this.is404Error(error), // also retry 404 error since AAD need sometime to sync created AAD app data
```

Comment:
Do we need to retry 404 error here? If not, you can just remove the axios-retry config since there's default retry policy configured in constructor.

Reply:
As I remember we have telemetry that collaboration apis retried due to 404 in v4, I prefer to keep it as it will not break user experience.